tests/check_if_exists_without_connect.phpt

```diff
@@ -8,6 +8,17 @@ if (!extension_loaded('zookeeper')) {
 --FILE--
 <?php
 $client = new Zookeeper();
-$client->exists('/test1');
+try {
```

Comment:
Does `--EXPECT--` part support PHP code?

like:

```php
--EXPECT--
Zookeeper->connect() was not called
<?php echo Zookeeper::CONNECT_NOT_CALLED; ?>
```

I'm worried about future changing of constant value.

Is this my over-worriness?